examples/metrics/grafana-dashboards/strimzi-kafka-bridge.json

```diff
@@ -652,7 +652,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "count(strimzi_bridge_kafka_consumer_last_heartbeat_seconds_ago != -1)",
+          "expr": "sum(strimzi_bridge_kafka_consumer_last_heartbeat_seconds_ago != bool -1)",
```

Comment:
Can you explain this one for me?

Reply:
This comes from #5889 - this really just copies the changes we did since the 0.26.0 release to the main installation and examples files.